src/main/java/org/javacord/bot/commands/InfoCommand.java

```diff
@@ -0,0 +1,28 @@
+package org.javacord.bot.commands;
+
+import de.btobastian.sdcf4j.Command;
+import de.btobastian.sdcf4j.CommandExecutor;
+import org.javacord.api.DiscordApi;
+import org.javacord.api.Javacord;
+import org.javacord.api.entity.channel.TextChannel;
+import org.javacord.api.entity.message.embed.EmbedBuilder;
+
+public class InfoCommand implements CommandExecutor {
+
+    @Command(aliases = "!info", async = true)
+    public void handleCommand(TextChannel channel) {
+        final DiscordApi api = channel.getApi();
+        EmbedBuilder embed = new EmbedBuilder()
+                .setTitle(api.getYourself().getName() + " - Official Javacord Bot")
+                .setThumbnail(api.getYourself().getAvatar())
+                .setDescription("The official bot for the Javacord Server.\n\n" +
```

Comment:
And probably no dot, this is not a sentence, but more a sub-caption
```suggestion
                .setDescription("The official bot for the Javacord Server\n\n" +
```